### PR TITLE
Update 07_Code_of_conduct.md

### PR DESCRIPTION
**Pull request by SpiritLevel**
Typos, grammar, rewording.

### CODE REVIEW

**Comment by willmorgan** [comment]
Hmm, shouldn't you be editing a novel, or something? :wink:

**Comment by SpiritLevel** [comment]
Just practising. I'll start on code in the near future ;)

**Comment by willmorgan** [comment]
Ha! Well, I think it's safe to say your editing skills are appreciated. The prose flows much better now.

File: docs/en/05_Contributing/07_Code_of_conduct.md [hunk 1]
```diff
@@ -10,39 +10,39 @@ Honour the code of conduct whenever you participate formally and informally in o
...
- * Whenever possible reference your comments to others with example code or links to documentation. This helps people learn and become more experienced community members and developers.
- * If you manage to solve your own problem, tell others how you solved it. This will help people in the future that find they have the same problem as you.
- * If you are reducing your input and potentially stepping away from the community please remember others might rely on your contributions. Be prepared to ensure these are kept open and available, spend some time handing over to another community member or core contributor to help continuity of SilverStripe open source.
+ * Whenever possible, reference your comments to others with example code or links to documentation. This helps people learn and become more experienced community members and developers.
+ * If you manage to solve your own problem, tell others how you solved it. This will help people in the future who have to solve the sort of problem.
```
Comment (dhensby) [inline docs/en/05_Contributing/07_Code_of_conduct.md]: Who have to solve the *same* sort of problem

Comment (dhensby) [inline docs/en/05_Contributing/07_Code_of_conduct.md]: Or just a similar problem

**Pull request merged**